compiler/src/main/kotlin/io/spine/protodata/ContextExtension.kt

```diff
@@ -24,14 +24,17 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
-package io.spine.protodata.cli.given
+package io.spine.protodata
 
-import io.spine.protodata.TypeEntered
-import io.spine.protodata.subscriber.CodeEnhancement
-import io.spine.protodata.subscriber.Subscriber
+import io.spine.server.BoundedContextBuilder
 
-class TestSubscriber : Subscriber<TypeEntered>(TypeEntered::class.java) {
+/**
+ * An extension above the `ProtoData` bounded context.
+ */
+public interface ContextExtension {
```

Comment:
Since this is a part of public API, let's put more attention to the name of this thing. The issue with the current name I see is that it exposes details that we want to hide from a user who's new to the DDD.

Even if this is an extension to the context, it may be exposed under a more common name, like `Plugin` or something.